planner/core/plan_cost_ver2.go

```diff
@@ -35,9 +35,14 @@ func GetPlanCost(p PhysicalPlan, taskType property.TaskType, option *PlanCostOpt
 	return getPlanCost(p, taskType, option)
 }
 
+var GenPlanCostTrace func(p PhysicalPlan, costV costVer2, taskType property.TaskType, option *PlanCostOption) 
```

Comment:
Please add some comments for this hook function.

Reply:
OK